src/dmd/hdrgen.d

```diff
@@ -2863,6 +2864,8 @@ extern (D) string trustToString(TRUST trust) pure nothrow
         return "@trusted";
     case TRUST.safe:
         return "@safe";
+    case TRUST.live:
+        return "@live";
```

Comment:
?

Reply:
One of those inconsistencies I was talking about.

Reply:
This is a bug fix, while moving the enums is a refactoring. You're the first to complain when those are mixed together.

Reply:
The two previous definitions of the enum were inconsistent. Something had to change. I chose this method to resolve them because it is additive, and will not break anything. At worst, it would be unused code.

I put off dealing with the ENUMTY inconsistency, because that was risky.

Reply:
More to the point, it was *not* a bug fix.

Reply:
This specific change is a change of behaviour, not a refactoring.

Reply:
No, it is not. It is the result of merging two different enums with the same name.

Reply:
Then let's fix ASTBase first, as it is in the wrong #12747

Reply:
> No, it is not. It is the result of merging two different enums with the same name.

Two enums which were assumed to be the same.
Looking at the commit that introduced it (https://github.com/dlang/dmd/commit/eaf864e8490f6544de144529ae22a4cfaaa531b5), I think that `live` should just be removed from `TRUST`, since it's unused AFAICS (as @ibuclaw noted).